### ISSUE TITLE
Prostate Specific Antigen testing

### ISSUE THREAD

**Issue by PedroOliveira28** [opensafely/codelist-development#137]
Good evening.

Looking for discussion and eventual sign off of the code list definition for PSA testing which I have (re-created) here:

https://www.opencodelists.org/codelist/user/poliveira28/psa-test/437eb36a/

@brianmackenna @HelenCEBM , I have assigned you both and myself for this to avoid the ill faith of my previous attempt of signing off my code list (which I just closed). By all means feel free to indicate someone who will be well placed to provide input on this. Many thanks.

**Comment by HelenCEBM** [comment]
Hi Pedro,

Thanks for including a thorough description of your methodology - very helpful! The search strategy seems sensible - but looking at the [tree](https://www.opencodelists.org/codelist/user/poliveira28/psa-test/437eb36a/#tree) I'm not sure why some of the codes are excluded that look similar to included codes. Was this intentional?

Thanks!

**Comment by PedroOliveira28** [comment]
Good morning, Helen.

This is my first attempt at creating a code list, and in this case this one is for dealing with a procedure (a diagnostic). I checked a few existing code lists. One of the closest ones was cholesterol testing:

https://www.opencodelists.org/codelist/opensafely/cholesterol-tests/09896c09/#tree

I was not sure myself if the following codes should be included in my tree:

Measurement of free prostate specific antigen in serum specimen (909341000000103)
Measurement of prostate specific antigen in serum specimen (909321000000105)
Measurement of prostate specific antigen using ultrasensitive assay (443174008)

In the cholesterol testing tree Cholesterol level and its sub-nodes total cholesterol level are included, whereas sub-nodes maximum and minimum cholesterol level are not. But going back to the description in the metadata I can see that it says "A SNOMED CT codelist describing primary care requests for **total** cholesterol." Do you think that adding the three codes above to those included in the current tree will solve the issue? Or do you see other problems? Many thanks.

**Comment by HelenCEBM** [comment]
Yes, I would include those three codes you list - I think we will want to make sure we have included every possible code that patients could have to avoid under-counting.  There are a few others which look like "total PSA" (or just "PSA") in combination with "level", "measurement" or "concentration" (see ticked items below), which I think should also be included. We can exclude anything different to this (see crossed items). But do feel free to say if you disagree with any of these - we can simply record the reasons for exclusion here.

(Given that we believe the total level should always be reported, it shouldn't be necessary to include anything relating to "ratio" or "raised"/"normal"/"abnormal". I'm assuming that "mass concentration" is equivalent to "total PSA level" but this may not be the case! I'm also assuming it doesn't matter whether it specifies that PSA was measured in "serum", "plasma" or neither - from other tests we have looked at these seem to be equivalent, at least for our purposes here). 

![image](https://user-images.githubusercontent.com/24392156/133777980-87f8f353-e847-4fef-9d21-e443e03de66f.png)

Does that look sensible?

**Comment by PedroOliveira28** [comment]
Thank you, Helen. This is extremely helpful.

Taking into account your comments and my revision of the code list resulting from the initial search, I think I need to add(include) / take out / leave (as per comments below before each code - no comment for leave) the following codes:

Free prostate specific antigen level (1030021000000101)
Free prostate specific antigen level (392837002)
Free prostate specific antigen level (393792009)
Free prostate specific antigen level (395145001)
[INCLUDE] Mass concentration of prostate specific antigen in plasma (1107801000000104)
[INCLUDE] Mass concentration of prostate specific antigen in serum (1107811000000102)
[INCLUDE] Measurement of free prostate specific antigen in serum specimen (909341000000103)
[INCLUDE] Measurement of prostate specific antigen in serum specimen (909321000000105)
[INCLUDE] Measurement of prostate specific antigen using ultrasensitive assay (443174008)
[INCLUDE] Prostate specific antigen (& serum level) (166158002)
Prostate specific antigen level (1030791000000100)
Prostate specific antigen measurement (63476009)
Prostate-specific antigen level (273968004)
[INCLUDE] Quantitative measurement of mass concentration of prostate specific antigen in serum or plasma specimen (443969004)
[INCLUDE] Serum free prostate specific antigen level (1000481000000100)
Serum free prostate specific antigen level (909351000000100)
[INCLUDE] Serum prostate specific antigen level (1000381000000105)
Serum prostate specific antigen level (909331000000107)
Total prostate specific antigen level (1006591000000104)
Total prostate specific antigen level (377981000000102)
Total prostate specific antigen level (377991000000100)
Total prostate specific antigen level (378001000000109)
Ultra-sensitive prostate specific antigen level (492741000000109)
Ultra-sensitive prostate specific antigen level (492751000000107)
Ultra-sensitive prostate specific antigen measurement (492731000000100)
[TAKE OUT] Prostate specific antigen abnormal (166160000)
[TAKE OUT] Prostate specific antigen normal (166159005)

This will just leave out codes that not mention level/concentration measurement (e.g. monitoring, threshold) or do not have any "qualifier" (e.g. Prostate specific antigen).

Please let me know what you think. I think we are very close to the solution. Again many thanks for your help.

Pedro

**Comment by PedroOliveira28** [comment]
Thank you. Helen. Before closing the issue and opening a new code list on PSA, can I ask @brianmackenna if he has comments, he would be happy to sign this off after revision, or would like to indicate someone better placed to do sign off? Apologies, here I went for the people I had had discussions around Open Safely, without being sure if each could provide the two different types of sign off required. Many thanks!

**Comment by HelenCEBM** [comment]
Hi Pedro, we would suggest that the second sign off could come from your team, as it's important that you as a research team are happy with the codes.

**Comment by PedroOliveira28** [comment]
Thank you, Helen.

I had understood initially that for sign off of the code list both the epidemiological expert (on which capacity you are acting) and the clinical expert would need to be members of the Open Safely community to provide sign off. We are working under the direction of a clinical adviser in NHSE/I, who would be the obvious port of call to sign off the code list. 

Thinking about processes: would it be sufficient to email our clinical adviser asking to validate the list, and forward you the email with his approval so that it is on the record and the code list can be signed off? There may be other simpler ways of doing it, but this one looks like a sensible way forward, unless it is not admitted. Many thanks.

**Comment by PedroOliveira28** [comment]
Thank you, Helen. I will ask the clinical adviser if he is happy to have his comments reproduced here.

**Comment by brianmackenna** [comment]
H @PedroOliveira28 we won't have time this week to work on the list. We generally don't participate in the validation of codelists for onboarders as this is best done by the researchers working on the study and know the nuance of what exactly they are studying.

You can always press on with the list as is and get more checks on the codelist at a later date.
